index.html

```diff
@@ -1460,7 +1460,7 @@ <h2 id="docconformance">
                 (If not a descendant of an `article`, `aside`, `main`, `nav`
                 or `section` element, or an element with `role=article`, `complementary`,
                 `main`, `navigation` or `region`,
-                then <code>role=<a href="#index-aria-contentinfo">contentinfo</a></code>
+                then <code>role=<a href="#index-aria-contentinfo">banner</a></code>
```

Comment:
Need to change the link destination as well
```suggestion
                then <code>role=<a href="#index-aria-banner">banner</a></code>
```